app.js

```diff
@@ -1,34 +1,32 @@
-
-const path = require('path');
-
+const mongoose = require('mongoose');
 const express = require('express');
 const bodyParser = require('body-parser');
 
-const users = require('./routes/users.js');
-const cards = require('./routes/cards');
+const routes = require('./routes/routes');
```

Comment:
Можно лучше: точку входа в группу модулей именовать `index.js`.
`index.js` является базовым файлом, который будет подключен по-умолчанию, если будет произведено подключение модуля. Например, если переименовать, то можно подключить роуты вот так:
```suggestion
const routes = require('./routes');
```

Как минимум стало короче.